client/verta/verta/_pipeline_orchestrator/_orchestrator.py

```diff
@@ -134,14 +200,10 @@ def run(
         return self._outputs[step_name]
 
 
-class DeployedOrchestrator(_OrchestratorBase):  # TODO
-    """Inference pipeline orchestrator using HTTP server models."""
```

Comment:
This is moved to the orchestrator service (https://github.com/VertaAI/python-services/pull/222).